packages/expo-av/android/src/main/java/expo/modules/av/player/datasource/CustomHeadersOkHttpDataSourceFactory.java

```diff
@@ -24,13 +24,16 @@
   private final TransferListener mListener;
   @Nullable
   private final CacheControl mCacheControl;
+  @Nullable
+  private final TransferListener mBandwidthMeter;
 
-  public CustomHeadersOkHttpDataSourceFactory(@NonNull Call.Factory callFactory, @Nullable String userAgent, @Nullable Map<String, Object> requestHeaders) {
+  public CustomHeadersOkHttpDataSourceFactory(@NonNull Call.Factory callFactory, @Nullable String userAgent, @Nullable Map<String, Object> requestHeaders, @Nullable TransferListener bandwidthMeter) {
     super();
     mCallFactory = callFactory;
     mUserAgent = userAgent;
     mListener = null;
     mCacheControl = null;
+    mBandwidthMeter = bandwidthMeter;
```

Comment:
There already is `mListener` field which is unused, could you please use that one?

Reply:
done 👍